src/components/VmDetails/cards/UtilizationCard/DiskCharts.js

```diff
@@ -19,15 +18,45 @@ import style from './style.css'
 
 import NoLiveData from './NoLiveData'
 
+const DiskBar = ({ path, total, used }) => {
+  const { unit, value } =
+    convertValueMap(
+      'B',
+      {
+        total: total,
+        used: used,
+      })
+  const thresholdError = 90
+  const thresholdWarning = 70
+  const usedInPercent = round(used / total * 100, 0)
+  return <div className={style['disk-fs-box']}>
+    <div className={style['disk-fs-name']}>{path}</div>
+    <div className={style['disk-fs-bar']}>
+      <UtilizationBar
+        now={usedInPercent}
+        thresholdWarning={thresholdWarning}
+        thresholdError={thresholdError}
+      />
+    </div>
+    <div className={style['disk-fs-used']}><strong>{round(value.used, 0)} of {round(value.total, 0)} {unit}</strong> Used</div>
```

Comment:
The `strong` should be in the style instead ... `font-weight: bold;`

Reply:
In this case I would prefer `strong`, because it will bold only some part of text, it is more readable, then adding a lot of `span`s and classes. Also this style would work for localization.

Reply:
Good point.